code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -48,6 +48,7 @@ Difficulty: Hard
 	var/charging = FALSE
 	medal_type = BOSS_MEDAL_BUBBLEGUM
 	score_type = BUBBLEGUM_SCORE
+	bloodcrawl = BLOODCRAWL_EAT
```

Comment:
Mark your changes ![like this](https://github.com/yogstation13/tg-rebase/blob/master/yogstation/README.md)